### PR TITLE
Release v0.6.1

### PR DESCRIPTION
**Pull request by schmmd**

### CODE REVIEW

File: allennlp/version.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 _MAJOR = "0"
 _MINOR = "6"
-_REVISION = "1-unreleased"
+_REVISION = "2-unreleased"
```
Comment (brendan-ai2) [inline allennlp/version.py]: What's the significance of "unreleased" here?

Comment (schmmd) [reply allennlp/version.py]: @brendan-ai2 after the release is cut we set it to an unreleased version, so if you pip install from master it's clear you didn't install a released version.  The tag should be associated with the previous commit (although I'll verify that).

Comment (schmmd) [reply allennlp/version.py]: Yup--the correct commit is tagged.

Comment (schmmd) [reply allennlp/version.py]: https://github.com/allenai/allennlp/pull/1698/commits/4920249af097d1557be6ece3e5ce80d1f6ea0333#diff-e66fadcdcab33b8d2e449e323706e936R3

Comment (brendan-ai2) [reply allennlp/version.py]: Cool. Thanks for explaining :)

**Pull request merged**